### PR TITLE
Update ci-viewer.members.txt

### PR DESCRIPTION
**Pull request by Tabrizian**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/internal-acls/173)

### CODE REVIEW

File: ci-viewer.members.txt [hunk 1]
```diff
@@ -4,3 +4,4 @@ gaoce@caicloud.io
 hejinchi@cn.ibm.com
 holden.karau@gmail.com
 redhat-team@kubeflow.org
+iman.tabrizian@gmail.com
```
Comment (jlewi) [inline ci-viewer.members.txt]: alphabetical order please.

**Comment by Tabrizian** [comment]
@jlewi Sorry, I fixed the order.

**Pull request merged**